src/Public/Authentication.ps1

```diff
@@ -1369,6 +1374,7 @@ Add-PodeAuthSession -Name 'SessionAuth' -FailureUrl '/login'
 #>
 function Add-PodeAuthSession {
     [CmdletBinding(DefaultParameterSetName = 'Groups')]
+    [OutputType([hashtable])]
```

Comment:
This function doesn't return anything, so there's no OutputType needed

Reply:
same issue the internal scriptblock return an hashtable.
I'm fixing it